### PR TITLE
Add prometheus and rename

### PR DESCRIPTION
**Pull request by yutongz**
1. Add prometheus task to get ci metrics
2. rename manager to pilot

### CODE REVIEW

**Comment by apelisse** [comment]
@k8s-bot ok to test

File: velodrome/config.yaml [hunk 1]
```diff
@@ -120,13 +120,23 @@ projects:
...
+    prometheus:
+      prometheus-config.yml: |
+        global:
+          scrape_interval: 15m
+        scrape_configs:
+          - job_name: istio-ci-metrics
+            metrics_path: /metrics
+            static_configs:
+              - targets:
+                  - http://104.197.146.56:9103
```
Comment (apelisse) [inline velodrome/config.yaml]: So is that the definitive IP?

Comment (yutongz) [inline velodrome/config.yaml]: Yes, I will not change it unless it's really necessary

**Comment by fejta** [comment]
What's up with merging PRs that fail unit tests?

**Comment by apelisse** [comment]
The test is already broken, this PR didn't break anything new. Reverting is
not helping. I ran the test locally before merging to confirm.

**Pull request merged**